docs/document-level-security/CONFLUENCE.md

```diff
@@ -0,0 +1,27 @@
+### Setting up the Confluence connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document level security (DLS) enables you to restrict access to documents based on a user’s permissions. This feature is available by default for the Confluence connector.
+
+Refer to [document level security](https://www.elastic.co/guide/en/enterprise-search/master/dls.html) for more information.
+
+**Note:** Refer to [DLS in Search Applications](https://www.elastic.co/guide/en/enterprise-search/master/dls-e2e-guide.html) to learn how to ingest data from Confluence with DLS enabled, when building a search application.
+
+## Additional Pre-requisite
+
+- To access user data on Jira Administration, the account that you created needs to be granted a **Product Access** for `Jira Administration` with `Product Admin` access by an administrator from [Atlassian Admin](http://admin.atlassian.com/).
```

Comment:
Should this bullet point refer to "Confluence Administration" instead of "Jira Administration"?

Reply:
No, Internally all `Atlassian` products uses `Jira Administration` for users(`Atlassian`) management stuff.